src/Rector/ValueObject/FunctionToStaticConfiguration.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace DrupalRector\Rector\ValueObject;
+
+class FunctionToStaticConfiguration {
+
+    protected string $deprecatedFunctionName = 'file_directory_os_temp';
+
+    protected string $className = 'Drupal\Component\FileSystem\FileSystem';
+
+    protected string $methodName = 'getOsTemporaryDirectory';
```

Comment:
Should this defaults exist?

Reply:
yeah it shouldnt, those leaked... will fix.

Reply:
Removed the default from the confguration.